conda-forge.yml

```diff
@@ -24,3 +24,4 @@ provider:
   osx: github_actions
   win: github_actions
 test_on_native_only: true
+upload_on_branch: main
```

Comment:
Configured this to only upload packages on the `main` branch. That way if we push to other branches on this feedstock (like was done with this PR), uploads won't occur (that is not until that branch is merged into `main`)